assets/tld.json

```diff
@@ -1 +0,0 @@
H","RMIT","ROOM","RSVP","RUHR","SAFE","SALE","SARL","SAVE","SAXO","SCOR","SCOT","SEAT","SEEK","SEXY","SHAW","SHIA","SHOP","SHOW","SILK","SINA","SITE","SKIN","SNCF","SOHU","SONG","SONY","SPOT","STAR","SURF","TALK","TAXI","TEAM","TECH","TEVA","TIAA","TIPS","TOWN","TOYS","TUBE","VANA","VISA","VIVA","VIVO","VOTE","VOTO","WANG","WEIR","WIEN","WIKI","WINE","WORK","XBOX","YOGA","ZARA","ZERO","ZONE","exit","zkey","AAA","ABB","ABC","ACO","ADS","AEG","AFL","AIG","ANZ","AOL","APP","ART","AWS","AXA","BAR","BBC","BBT","BCG","BCN","BET","BID","BIO","BIZ","BMS","BMW","BNL","BOM","BOO","BOT","BOX","BUY","BZH","CAB","CAL","CAM","CAR","CAT","CBA","CBN","CBS","CEB","CEO","CFA","CFD","COM","CRS","CSC","DAD","DAY","DDS","DEV","DHL","DIY","DNP","DOG","DOT","DTV","DVR","EAT","ECO","EDU","ESQ","EUS","FAN","FIT","FLY","FOO","FOX","FRL","FTR","FUN","FYI","GAL","GAP","GDN","GEA","GLE","GMO","GMX","GOO","GOP","GOT","GOV","HBO","HIV","HKT","HOT","HOW","IBM","ICE","ICU","IFM","INC","ING","INK","INT","IST","ITV","JCB","JCP","JIO","JLC","JLL","JMP","JNJ","JOT","JOY","KFH","KIA","KIM","KPN","KRD","LAT","LAW","LDS","LLC","LOL","LPL","LTD","MAN","MAP","MBA","MED","MEN","MIL","MIT","MLB","MLS","MMA","MOE","MOI","MOM","MOV","MSD","MTN","MTR","NAB","NBA","NEC","NET","NEW","NFL","NGO","NHK","NOW","NRA","NRW","NTT","NYC","OBI","OFF","ONE","ONG","ONL","OOO","ORG","OTT","OVH","PAY","PET","PHD","PID","PIN","PNC","PRO","PRU","PUB","PWC","QVC","RED","REN","RIL","RIO","RIP","RUN","RWE","SAP","SAS","SBI","SBS","SCA","SCB","SES","SEW","SEX","SFR","SKI","SKY","SOY","SRL","SRT","STC","TAB","TAX","TCI","TDK","TEL","THD","TJX","TOP","TRV","TUI","TVS","UBS","UNO","UOL","UPS","VET","VIG","VIN","VIP","WED","WIN","WME","WOW","WTC","WTF","XIN","XXX","XYZ","YOU","YUN","ZIP","bit","gnu","i2p","AC","AD","AE","AF","AG","AI","AL","AM","AO","AQ","AR","AS","AT","AU","AW","AX","AZ","BA","BB","BD","BE","BF","BG","BH","BI","BJ","BM","BN","BO","BR","BS","BT","BV","BW","BY","BZ","CA","CC","CD","CF","CG","CH","CI","CK","CL","CM","CN","CO","CR","CU","CV","CW","CX","CY","CZ","DE","DJ","DK","DM","DO","DZ","EC","EE","EG","ER","ES","ET","EU","FI","FJ","FK","FM","FO","FR","GA","GB","GD","GE","GF","GG","GH","GI","GL","GM","GN","GP","GQ","GR","GS","GT","GU","GW","GY","HK","HM","HN","HR","HT","HU","ID","IE","IL","IM","IN","IO","IQ","IR","IS","IT","JE","JM","JO","JP","KE","KG","KH","KI","KM","KN","KP","KR","KW","KY","KZ","LA","LB","LC","LI","LK","LR","LS","LT","LU","LV","LY","MA","MC","MD","ME","MG","MH","MK","ML","MM","MN","MO","MP","MQ","MR","MS","MT","MU","MV","MW","MX","MY","MZ","NA","NC","NE","NF","NG","NI","NL","NO","NP","NR","NU","NZ","OM","PA","PE","PF","PG","PH","PK","PL","PM","PN","PR","PS","PT","PW","PY","QA","RE","RO","RS","RU","RW","SA","SB","SC","SD","SE","SG","SH","SI","SJ","SK","SL","SM","SN","SO","SR","ST","SU","SV","SX","SY","SZ","TC","TD","TF","TG","TH","TJ","TK","TL","TM","TN","TO","TR","TT","TV","TW","TZ","UA","UG","UK","US","UY","UZ","VA","VC","VE","VG","VI","VN","VU","WF","WS","YE","YT","ZA","ZM","ZW"]
```

Comment:
Should probably not be deleted?

Reply:
This file is generated on build (scripts/tld.js) and was already included in the gitignore file.
So I figured it was there by accident and would only generate noise.